### ISSUE TITLE
Does not work with uppercase color codes

### ISSUE THREAD

**Issue by ShiChenCong** [vimpostor/vim-tpipeline#26]
Thank you for the work! 

in normal mode, status line works fine
<img width="1438" alt="image" src="https://user-images.githubusercontent.com/22486446/173726513-5ca0085a-0b95-4244-8d4f-3851949a6b94.png">
in command mode, status line background color disappear
<img width="1430" alt="image" src="https://user-images.githubusercontent.com/22486446/173726524-cda40d12-bf4c-4de9-aa21-a494e8078ca9.png">

**Comment by ShiChenCong** [comment]
only happens when `theme = "catppuccin"`,maybe it's colorscheme's issue?

**Comment by ShiChenCong** [comment]
it's catppuccin's issue

**Comment by vimpostor** [comment]
> it's catppuccin's issue

If the statusline embedded in tmux looks different than from what it would look like without this plugin, then it's a bug in this plugin.
What statusline plugin are you using? You only told me the theme (catppuccin), so I can't reproduce. Try taking a screenshot with both statuslines next to each other by using `:set laststatus=2`.

**Comment by vimpostor** [comment]
Can you please give your lualine and catppuccin config? I can't reproduce with the defaults

**Comment by ShiChenCong** [comment]
```lua
local status, lualine = pcall(require, "lualine")
if (not status) then return end
lualine.setup {
  options = {
    icons_enabled = true,
    theme = "catppuccin",
    globalstatus = true,
    section_separators = { left = '', right = '' },
    component_separators = { left = '', right = '' },
    disabled_filetypes = {}
  },
  sections = {
    lualine_a = { 'mode' },
    lualine_b = { 'branch' },
    lualine_c = {
      { 'diagnostics', sources = { "nvim_diagnostic" }, symbols = { error = ' ', warn = ' ', info = ' ', hint = ' ' } },
    },
    lualine_x = {
      'encoding',
      'filetype'
    },
    lualine_y = { 'progress' },
    lualine_z = { 'location' }
  },
  inactive_sections = {
    lualine_c = { {
      'filename',
      file_status = true, -- displays file status (readonly status, modified status)
      path = 1 -- 0 = just filename, 1 = relative path, 2 = absolute path
    } },
  },
  tabline = {},
  extensions = { 'fugitive', 'fzf', 'nvim-tree', 'quickfix', 'toggleterm' }
}
```
```lua
local catppuccin = require("catppuccin")
catppuccin.setup({
  integrations = {
    bufferline = false,
    nvimtree = {
      enabled = true,
      show_root = false,
      transparent_panel = false,
    },
    gitsigns = true,
    telescope = true,
    cmp = true,
    treesitter = true,
    native_lsp = {
      enabled = true,
      virtual_text = {
        errors = "italic",
        hints = "italic",
        warnings = "italic",
        information = "italic",
      },
      underlines = {
        errors = "underline",
        hints = "underline",
        warnings = "underline",
        information = "underline",
      },
      hop = true
    },
    lsp_trouble = false,
  }
})
vim.g.catppuccin_flavour = "macchiato" -- latte, frappe, macchiato, mocha
vim.cmd [[colorscheme catppuccin]]
vim.cmd [[
  hi DiffAdd ctermbg=4 guibg=#283b4D guifg=NONE
  hi DiffDelete ctermfg=12 ctermbg=6 guibg=#3F2D3D  guifg=NONE
  hi DiffChange ctermbg=5 guibg=#272D43 guifg=NONE
  hi DiffText cterm=bold ctermbg=9 guibg=#394b70 guifg=NONE
  hi Folded guibg=#2c313a
]]

```

**Comment by ShiChenCong** [comment]
I see

**Comment by ShiChenCong** [comment]
I see,thank you 😄

**Comment by ShiChenCong** [comment]
Thank you for your patience😄,  After add this option, works fine

**Comment by vimpostor** [comment]
BTW this issue is now also fixed in tmux 3.4, I have implemented support for uppercase color codes in tmux: https://github.com/tmux/tmux/issues/3239
